gitpod-io/gitpod
# Replace /workspaces → /projects as default landing page for both users and teams

## Description


## Related Issue(s)

Fixes https://github.com/gitpod-io/gitpod/issues/5050

## How to test


1. Visiting the dashboard should open the Projects page by default
2. Switching to a team should open the Projects page by default

## Release Notes

```release-note
NONE
```
